scripts/simple.sh

```diff
@@ -0,0 +1,168 @@
+  --abi-path $ABI_PATH \
+  --aurora-secret-key 611830d3641a68f94a690dcc25d1f4b0dac948325ac18f6dd32564371735f32c || error_exit
+sleep 1
+result=$(aurora-cli --engine $ENGINE_ACCOUNT view-call -a 0x4cf003049d1a9c4918c73e9bf62464d904184555 -f value \
+  --abi-path $ABI_PATH || error_exit)
+assert_eq "$result" "6"
+sleep 1
+aurora-cli --engine $ENGINE_ACCOUNT call -a 0x4cf003049d1a9c4918c73e9bf62464d904184555 -f decrement \
+  --abi-path $ABI_PATH \
+  --aurora-secret-key 611830d3641a68f94a690dcc25d1f4b0dac948325ac18f6dd32564371735f32c || error_exit
+sleep 1
+result=$(aurora-cli --engine $ENGINE_ACCOUNT view-call -a 0x4cf003049d1a9c4918c73e9bf62464d904184555 -f value \
+  --abi-path $ABI_PATH || error_exit)
+assert_eq "$result" "5"
+sleep 1
+
+# Check read operations.
+aurora-cli --engine $ENGINE_ACCOUNT get-chain-id || error_exit
+aurora-cli --engine $ENGINE_ACCOUNT get-version || error_exit
+aurora-cli --engine $ENGINE_ACCOUNT get-owner || error_exit
+aurora-cli --engine $ENGINE_ACCOUNT get-bridge-prover || error_exit
+aurora-cli --engine $ENGINE_ACCOUNT get-balance 0x04b678962787ccd195a8e324d4c6bc4d5727f82b || error_exit
+aurora-cli --engine $ENGINE_ACCOUNT get-code 0xa3078bf607d2e859dca0b1a13878ec2e607f30de || error_exit
+aurora-cli key-pair --seed 1 || error_exit
+block_hash=$(aurora-cli --network mainnet get-block-hash 88300374 || error_exit)
+assert_eq "$block_hash" "0xd31857e9ce14083a7a74092b71f9ac48b8c0d4988ad40074182c1f0ffa296ec5"
+
+# Register a new relayer address
+aurora-cli --engine $ENGINE_ACCOUNT register-relayer 0xf644ad75e048eaeb28844dd75bd384984e8cd508 || error_exit
+sleep 1
+
+# Set a new owner. The functionality has not been released yet.
+#aurora-cli --engine $ENGINE_ACCOUNT set-owner node0 || error_exit
+#sleep 1
+#owner=$(aurora-cli --engine $ENGINE_ACCOUNT get-owner || error_exit)
+#assert_eq "$owner" node0
+#export NEAR_KEY_PATH=$NODE_KEY_PATH
+#aurora-cli --engine $ENGINE_ACCOUNT set-owner aurora.node0 || error_exit
+#sleep 1
+#owner=$(aurora-cli --engine $ENGINE_ACCOUNT get-owner || error_exit)
+#assert_eq "$owner" $ENGINE_ACCOUNT
+
+# Check pausing precompiles. Not working on the current release because of
+# hardcoded aurora account in EngineAuthorizer.
+#export NEAR_KEY_PATH=$AURORA_KEY_PATH
+#mask=$(aurora-cli --engine $ENGINE_ACCOUNT paused-precompiles || error_exit)
+#assert_eq "$mask" 0
+#aurora-cli --engine $ENGINE_ACCOUNT pause-precompiles 1 || error_exit
+#sleep 1
+#mask=$(aurora-cli --engine $ENGINE_ACCOUNT paused-precompiles || error_exit)
+#assert_eq "$mask" 1
+#aurora-cli --engine $ENGINE_ACCOUNT pause-precompiles 2 || error_exit
+#sleep 1
+#mask=$(aurora-cli --engine $ENGINE_ACCOUNT paused-precompiles || error_exit)
+#assert_eq "$mask" 3
+#aurora-cli --engine $ENGINE_ACCOUNT resume-precompiles 3 || error_exit
+#sleep 1
+#mask=$(aurora-cli --engine $ENGINE_ACCOUNT paused-precompiles || error_exit)
+#assert_eq "$mask" 0
+
+# Stop NEAR node and clean up.
```

Comment:
It will be required after the next aurora engine release. And also, I decided to leave it as an example of that functionality which could be applied to the aurora engine from the develop branch. So it was made intentionally.

Reply:
By the way, if you insist, I will delete it.

Reply:
It has been resolved by releasing a new version of Aurora engine 😄